src/components/ComparisonTool/ComparisonModal.vue

```diff
@@ -94,90 +98,7 @@
                     v-model="radios"
                     mandatory
                     dense
-                  >
```

Comment:
I think Siobhan meant you could add a div here and add "None" so it isn't displayed, instead of deleting the code